src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Main {
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
+        for (int i = 0; i < figures.length; i++) {
+            if (i < figures.length / 2) {
+
+                figures[i] = new FigureSupplier().getRandomFigure();
+
+            } else {
+                figures[i] = new FigureSupplier().getDefaultFigure();
```

Comment:
Please don’t create a new FigureSupplier() inside the loop. Please remove empty lines. Firstly you should initialize an array and in the next loop you should print it.